__tests__/lottoCalculator.test.js

```diff
@@ -0,0 +1,93 @@
+import LottoCalculator from '../src/domains/LottoCalculator';
+
+describe('당첨 번호와 발행 번호 비교하는 테스트', () => {
+  test('당첨 번호와 발행 번호를 비교시 3개가 일치한다.', () => {
+    const lottoNumbers = {
+      winningNumbers: [1, 2, 3, 4, 5, 6],
+      bonusNumber: 7,
+    };
+
+    const generatedLottos = [[1, 2, 3, 7, 8, 9]];
+
+    const lottoCalculator = new LottoCalculator(lottoNumbers, generatedLottos);
+
+    const result = lottoCalculator.compare(
+      lottoNumbers.winningNumbers,
+      generatedLottos[0],
+    );
+
+    expect(result).toEqual(3);
+  });
+
+  test('보너스 번호가 일치하면 true를 반환한다.', () => {
+    const lottoNumbers = {
+      winningNumbers: [1, 2, 3, 4, 5, 6],
+
+      bonusNumber: 7,
```

Comment:
사소한데, 여기 개행은 조금 어색해보이네요, 객체에 property가 너무 많아서 그 의미단위로 구분하고자 함이 아니면 이렇게는 잘 안쓰는 것 같습니다.

```suggestion
      winningNumbers: [1, 2, 3, 4, 5, 6],
      bonusNumber: 7,
```